README.md

```diff
@@ -3,9 +3,9 @@ pythonnet
 
 Python for .NET is a package that gives Python programmers nearly seamless integration with the .NET Common Language Runtime (CLR) and provides a powerful application scripting tool for .NET developers. It allows Python code to interact with the CLR, and may also be used to embed Python into a .NET application.
 
-[![Build Status](https://travis-ci.org/pythonnet/pythonnet.png?branch=master)](https://travis-ci.org/pythonnet/pythonnet)
+[![Build Status](https://travis-ci.org/denfromufa/pythonnet.png?branch=develop)](https://travis-ci.org/denfromufa/pythonnet)
```

Comment:
Agreed. I think we should merge into master and do a 2.1 release as soon as
the 3.5 stuff is all working and tested, so the CI build statuses should be
left as they were.

On Mon, Feb 22, 2016 at 10:43 AM Benedikt Reinartz <notifications@github.com>
wrote:

> In README.md
> <https://github.com/pythonnet/pythonnet/pull/154#discussion_r53608999>:
>
> > @@ -3,9 +3,9 @@ pythonnet
> >
> >  Python for .NET is a package that gives Python programmers nearly seamless integration with the .NET Common Language Runtime (CLR) and provides a powerful application scripting tool for .NET developers. It allows Python code to interact with the CLR, and may also be used to embed Python into a .NET application.
> >
> > -[![Build Status](https://travis-ci.org/pythonnet/pythonnet.png?branch=master)](https://travis-ci.org/pythonnet/pythonnet)
> > +[![Build Status](https://travis-ci.org/denfromufa/pythonnet.png?branch=develop)](https://travis-ci.org/denfromufa/pythonnet)
>
> Setting branch to develop might make sense, but if you're pushing to the
> main repo you should keep the reference as it is ...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pythonnet/pythonnet/pull/154/files#r53608999>.
>